src/applications/disability-benefits/2346/components/UnverifiedPrefillAlert.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleLoginModal } from 'platform/site-wide/user-nav/actions';
+
+const UnverifiedPrefillAlert = props => (
+  <div className="usa-alert usa-alert-info schemaform-sip-alert">
+    <div className="usa-alert-body">
+      <h3 className="usa-alert-heading">
+        Save time—and save your work in progress—by signing in before starting
```

Comment:
Is this a typo? 🙂 
`Save time—and`
`Save time and`

Reply:
Not a typo - this matches the copy and also matches default copy by that component

Reply:
Hmm, do you have a content editor on your team? We should get someone from content to take a look at this as it's a little confusing imho 🙂